### ISSUE TITLE
`MultiSelect` component

### ISSUE THREAD

**Issue by adamkudrna** [react-ui-org/react-ui#574]
## Composition
 
- MultiSelect
	- MultiSelectInput
		- MultiSelectToggle
		- MultiSelectTag
	- MultiSelectDropdown
		- MultiSelectDropdownOption
		- MultiSelectDropdownHeader

**Comment by mbohal** [comment]
I think `MutliSelectField` is the correct name.

**Comment by adamkudrna** [comment]
Yes, since we have a `SelectField`, `MultiSelectField` is the correct name (for us).
